pkg/cmd/account/azure/list/list.go

```diff
@@ -62,23 +62,24 @@ func listAzureAccounts(client *client.Client, cmd *cobra.Command, s *spinner.Spi
 				Id                 string
 				Name               string
 				SubscriptionNumber string
-				TenantId           string
-				ClientId           string
+				AccountType        string
 				AzureEnvironment   string
 			}{
 				Id:                 acc.GetID(),
 				Name:               acc.GetName(),
 				SubscriptionNumber: acc.SubscriptionID.String(),
-				TenantId:           acc.TenantID.String(),
-				ClientId:           acc.ApplicationID.String(),
+				AccountType:        string(acc.AccountType),
 				AzureEnvironment:   acc.AzureEnvironment,
 			}
 		},
 		Table: output.TableDefinition[accounts.IAccount]{
-			Header: []string{"NAME", "SUBSCRIPTION ID", "TENANT ID", "APPLICATION ID", "AZURE ENV"},
+			Header: []string{"NAME", "SUBSCRIPTION ID", "AZURE ENV"},
```

Comment:
`"AZURE ENV"` ➡️ `"AZURE ENVIRONMENT"` 🤔  Maybe. Not sure about this one.

Reply:
picking AZURE ENVIRONMENT because ENVIRONMENT already exist on the accounts as Octopus ENVIRONMENTs